services/multisig-service/POSTGRES_DOCKER.md

```
@@ -0,0 +1,150 @@
+
+### 3. Truy cập PostgreSQL
+
+#### Cách 1: Vào psql shell (tương tác)
+```powershell
+docker exec -it postgres-db psql -U postgres -d multisig_db
+```
+
+Sau đó bạn có thể chạy các lệnh SQL:
+```sql
+\l              -- Liệt kê databases
+\dt             -- Liệt kê tables
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)